beat/heart.py

```diff
@@ -1606,6 +1607,50 @@ def get_velocity_model(
 
         source_model = cake.load_model(
             earth_model_name, crust2_profile=profile)
+            
+#######################################################################################
+    elif gfc.use_epcrust:
+        logger.info('Using EPcrust profile')
+        # load velocity profile from EPcrust0.5x0.5 and check for water
+        if gfc.epcrust_radius != 0:
+            profile = ep_crust.avg_profile(location.lat, location.lon, gfc.epcrust_radius)
+            logger.info('Average EPcrust profile in radius of %.f km around the coordinates: %.2f, %.2f' % (gfc.epcrust_radius, profile._lat, profile._lon))
+        else:
+            profile = ep_crust.get_profile(location.lat, location.lon)
+            logger.info('EPcrust profile on the coordinates: %.2f, %.2f' % (profile._lat, profile._lon))
+        if gfc.replace_water:
```

Comment:
can we reuse that functionality from the crust2 stuff? Thus could we create a function:

def remove_water from profile():

and use that in the crust2 and the ep_crust?